### PR TITLE
docs/getting-started.md

### PR DESCRIPTION
**Pull request by ph1p**
Need your feedback (:

### CODE REVIEW

File: content/docs/getting-started.md [hunk 1]
```diff
@@ -18,109 +18,125 @@ redirect_from:
...
+### Empfohlene Kurse {#recommended-courses}
 
-Sometimes people find third-party books and video courses more helpful than the official documentation. We maintain [a list of commonly recommended resources](/community/courses.html), some of which are free.
+Manchmal finden Personen Bücher und Videokurse von Drittanbietern hilfreicher als die offizielle Dokumentation, deshalb
+führen wir eine [Liste von empfehlenswerten Quellen](/community/courses.html). Einige sind sogar kostenlos verfügbar.
 
-### Advanced Concepts {#advanced-concepts}
+### Fortgeschrittene Konzepte {#advanced-concepts}
 
 Once you're comfortable with the [main concepts](#main-concepts) and played with React a little bit, you might be interested in more advanced topics. This section will introduce you to the powerful, but less commonly used React features like [context](/docs/context.html) and [refs](/docs/refs-and-the-dom.html).
```
Comment (jenbuzz) [inline content/docs/getting-started.md]: Missing translation?

Comment (ph1p) [reply content/docs/getting-started.md]: Oh ... I translated it on an other computer and forgot to push it

File: content/docs/getting-started.md [hunk 2]
```diff
@@ -18,109 +18,125 @@ redirect_from:
...
+Wenn du einen Designhintergrund hast, könnte [diese Sammlung](http://reactfordesigners.com/) sehr hilfreich für dich sein.
 
 ### JavaScript Resources {#javascript-resources}
 
-The React documentation assumes some familiarity with programming in the JavaScript language. You don't have to be an expert, but it's harder to learn both React and JavaScript at the same time.
+Die React Dokumentation setzt fundierte JavaScript Sprachkentnisse voraus. Du musst keine Experte sein, aber es wird dir viel schwerer fallen, React und JavaScript gleichzeitig zu lernen.
 
-We recommend going through [this JavaScript overview](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) to check your knowledge level. It will take you between 30 minutes and an hour but you will feel more confident learning React.
+Wir empfehlen dir diesen [JavaScript-Überblick](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) durchzugehen und deinen Wissensstand zu überprüfen. Es wird dich zwischen 30 Minuten und einer Stunde deiner Zeit
+kosten, aber danach solltest du dich sicherer fühlen, React zu lernen.
```
[the remark this answers is not in the document]
Comment (ph1p) [reply content/docs/getting-started.md]: This is an automatic line break

File: content/docs/getting-started.md (hunk 1 above)
Comment (ph1p) [reply content/docs/getting-started.md]: Added.

File: content/docs/getting-started.md [hunk 3]
```diff
@@ -18,109 +18,122 @@ redirect_from:
...
+Manchmal finden Personen Bücher und Videokurse von Drittanbietern hilfreicher als die offizielle Dokumentation, deshalb
+führen wir eine [Liste von empfehlenswerten Quellen](/community/courses.html). Einige sind sogar kostenlos verfügbar.
 
-### Advanced Concepts {#advanced-concepts}
+### Fortgeschrittene Konzepte {#advanced-concepts}
 
-Once you're comfortable with the [main concepts](#main-concepts) and played with React a little bit, you might be interested in more advanced topics. This section will introduce you to the powerful, but less commonly used React features like [context](/docs/context.html) and [refs](/docs/refs-and-the-dom.html).
+Wenn du einmal mit den Huaptkonzepten vetraut bist und ein wenig mit React herumgespielt hast, bist du vielleicht an den fortgeschrittenen Konzepten interessiert. Dieser Abschnitt zeigt dir mächtige React-Funktionen, wie [context](/docs/context.html) und [refs](/docs/refs-and-the-dom.html), diese finden aber weniger häufig Verwendung.
```
Comment (jenbuzz) [inline content/docs/getting-started.md]:
```suggestion
Wenn du einmal mit den Hauptkonzepten vertraut bist und ein wenig mit React herumgespielt hast, bist du vielleicht an den fortgeschrittenen Konzepten interessiert. Dieser Abschnitt zeigt dir mächtige React-Funktionen, wie [context](/docs/context.html) und [refs](/docs/refs-and-the-dom.html), diese finden aber weniger häufig Verwendung.
```

**Pull request merged**